Code/GraphMol/MarvinParse/MarvinParser.h

```diff
@@ -53,12 +53,14 @@ RDKIT_MARVINPARSER_EXPORT ChemicalReaction *MrvFileToChemicalReaction(
 
 RDKIT_MARVINPARSER_EXPORT void MolToMrvFile(const ROMol &mol,
                                             const std::string &fName,
-                                            bool includeStereo, int confId,
-                                            bool kekulize,
+                                            bool includeStereo = true,
+                                            int confId = -1,
+                                            bool kekulize = true,
```

Comment:
@bjonnh-work @tadhurst-cdd Is `kekulize = true` the correct behavior for the MRV writer or should it be writing aromatic bonds unless the user specifies otherwise?

Reply:
the format supports aromatic bonds, but I am sure the preference is for kekuled forms.   Of all the examples I have seem, they are all kekuled.